pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -12,6 +12,10 @@ fetchSubmodules=
 builder=
 branchName=$NIX_PREFETCH_GIT_BRANCH_NAME
 
+# ENV params
+out=${out:-}
+http_proxy=${http_proxy:-}
```

Comment:
actually shellcheck complains with [SC2154](https://github.com/koalaman/shellcheck/wiki/SC2154). Arguably it's a valid use of unassigned variables but I prefer not to annotate the code.